course_discovery/apps/api/v1/views/search.py

```diff
@@ -145,7 +145,7 @@ class CourseSearchViewSet(BaseHaystackViewSet):
 
 
 class CourseRunSearchViewSet(BaseHaystackViewSet):
-    ordering_fields = ('start', 'id', 'title')
+    ordering_fields = ('start', 'id', 'title_override')
```

Comment:
@iam-mhaseeb Why we needed a new field `title_override` instead of existing `title` field for ordering?

Reply:
Haystack restricts us from doing sorting on the char field we need to use a separate field with `stored=True and indexed=False`.
Here are details: https://github.com/django-haystack/django-haystack/issues/569#issuecomment-7371509

Reply:
Also, `title_override` is coming from CourseRun model which is better but `title` from coming from the Course model.

Reply:
What about `id` ? that we use to sort the latest

Reply:
It's coming from the `CourseRun` model but I suggest dropping that because that's a simple integer id.